app/components/case-studies-components/CaseStudiesTitleText.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+interface CaseStudiesTitleTextProps {
+  wordOne: string;
+  wordTwo: string;
+  wordThree: string;
+}
+
+const CaseStudiesTitleText: React.FC<CaseStudiesTitleTextProps> = ({
+  wordOne,
+  wordTwo,
+  wordThree,
```

Comment:
Be sure to be a little more descriptive than `wordOne`, `wordTwo`, `wordThree`...

Reply:
I have changed this now